modules/event/year-navigation.module.css

```diff
@@ -0,0 +1,25 @@
+.navigation {
+  width: 100%;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.25);
+  margin-bottom: 20px;
+}
+
+.list {
+  display: flex;
+  margin-top: 56px;
+  font-size: 16px;
+  font-weight: 700;
+  list-style: disc outside;
+  gap: 20px;
+  overflow: auto;
+  scroll-snap-type: x proximity;
```

Comment:
it seems to have no effect alone, you must add the property `scroll-snap-align` (`start` for instance) on the children `li`